src/esm_tests/repos.py

```diff
@@ -36,6 +43,24 @@ def update_resources_submodule(info, verbose=True):
             sys.exit(1)
 
 def check_resources(info, verbose=True):
+    """
+    Check if the submodule ``resources`` (``esm_tests_info``) is installed yet, and if
+    not, asks the user whether it needs to install it or not. If the user wants to
+    install it sets ``info["repo_update"] = True``.
+
+    Parameters
+    ----------
+    info : dict
+        Dictionary with the general info about the tests.
+    verbose : bool
+        Verbose option.
+
+    Notes
```

Comment:
I am unsure how difficult this is, but it might be interesting to create "custom" Sphinx headings. @denizural is correct, this heading should not be `Raises` as we do not raise an error here, but it is an `Error` that might be usefully separated out in the documentation to show what sort of things break this function.

Personally I would just raise an exception, but users don't like long scary tracebacks...

Reply:
We can always catch the exception with `user_error` to suppress the traceback & exit gracefully.